composer.json

```diff
@@ -149,7 +155,7 @@
     "nette/utils": "^2.5",
     "phpstan/phpstan": "^0.11",
     "symplify/easy-coding-standard-tester": "^5.2.17",
-    "symplify/changelog-linker": "^5.2.17",
+    "symplify/changelog-linker": "dev-rv-base-branch",
```

Comment:
This seems to be problematic on travis. Do you know why?

Reply:
I would publish this fork on Packagist as `shopsys/changelog-linker` in version `v5.5.0`. That should resolve this problem.